src/net/sourceforge/kolmafia/persistence/FaxBotDatabase.java

```diff
@@ -440,6 +449,20 @@ public void run() {
       FaxBotDatabase.faxbots.addAll(bots);
     }
 
+    private static final String NOHASH = "Problem with hash";
+
+    private static String computeHash(File file) {
+      String checksum;
+      try {
+        byte[] data = Files.readAllBytes(file.toPath());
+        byte[] hash = MessageDigest.getInstance("MD5").digest(data);
+        checksum = new BigInteger(1, hash).toString(16);
```

Comment:
Rather than parsing this as an integer, why not just output the hex representation, e.g. `HexFormat.of().formatHex(hash)`?

Reply:
Done